docs/install/lotus/join-a-network.md

```diff
@@ -83,7 +83,7 @@ lotus send <target> <amount>
 
 ## Configure your node's connectivity
 
-To effectively accept incoming storage & retrieval deals, your Lotus node needs to be accessible to other nodes on the network. To improve your connectivity, be sure to: 
+To effectively accept incoming storage & retrieval deals, your Lotus node needs to be accessible to other nodes on the network. To improve your connectivity, be sure to:
```

Comment:
This doesn't seem to be miner docs. Normal users do not need to worry about this.. and if this is for miners, it has about 10% of the info required for mining so it's not very useful.

Reply:
This was copied from the [Lotu.sh docs](https://lotu.sh/en+join-testnet). If you have updated docs to add in here please suggest them.